tox.ini

```diff
@@ -30,13 +30,34 @@ commands = pep8 --ignore E501 luigi test examples bin
 deps = autopep8
 commands = autopep8 --ignore E309,E501 -a -i -r luigi test examples bin
 
-[testenv:docsapi]
-deps = sphinx
-commands = sphinx-apidoc -o doc/api -T luigi
-
 [testenv:docs]
+# Build documentation using sphinx.
+# Call this using `tox -e docs`.
 deps =
   sqlalchemy
   Sphinx
   sphinx_rtd_theme
-commands = sphinx-build -b html -d {envtmpdir}/doctrees doc doc/_build/html
+commands =
+    # build API docs
+    sphinx-apidoc -o doc/api -T luigi
+
+    # sync README.rst file
+    cp README.rst doc/README.rst
```

Comment:
Can you delete doc/readme.rst since you're copying README.rst into doc/README.rst now?

Also Sphinx is throwing some errors. Could it be because you need to copy README.rst into doc/readme.rst (lower case) rather than doc/README.rst?

```Warning, treated as error:
/home/travis/build/spotify/luigi/doc/readme.rst:: WARNING: document isn't included in any toctree```